presto-docs/src/main/sphinx/release/release-0.122.rst

```diff
@@ -8,3 +8,16 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
+
+* Fix TypeUtils.hashPosition for structural types. Grouping by structural types
```

Comment:
This should be explained in terms of impact to the end user. We don't put implementation details in release notes